postgres/changelog.d/16990.added

```diff
@@ -0,0 +1 @@
+[DBMON-3691] set collect_wal_metrics to false will disable wal file metrics collection for all pg versions
```

Comment:
```suggestion
Set `collect_wal_metrics` to false will disable wal file metrics collection for all pg versions
```

Changelog are mostly for customer and they don't have access to our internal board so we should avoid adding our jira tickets in them